yt/python/yt/local/bin/yt_local

```diff
@@ -225,6 +225,11 @@ def add_start_subparser(subparsers):
     parser.add_argument("--cell-tag", type=int, default=1,
                         help="cluster cell tag")
 
+    parser.add_argument("--job-proxy-logging-mode", default="sandbox", 
+                        help="job proxy logging mode: sandbox, separate_directory")
```

Comment:
There is [choices](https://docs.python.org/3/library/argparse.html#choices) option for such situation